test/index.js

```diff
@@ -451,6 +451,18 @@ function fastUpdate(s, n) {
       assert(s1and2.end());
       assert.deepEqual(result, [12, 2, 4, 44, 1, 12, 2]);
     });
+    it('emmits all events from atomic update', function () {
+      var result = [];
+      var a = stream(1);
+      var b = stream([a], function() { return a() * 2; });
+      var c = stream([a], function() { return a() + 4; });
+      var d = flyd.merge(b, c);
+      stream([d], function () {
+        result.push(d());
+      });
+      a(2);
+      assert.deepEqual(result, [2, 5, 4, 6]);
```

Comment:
Also here I suppose that events will be emitted in the same order as streams provided to `flyd.merge`.
Should that be like that? Or maybe opposite order, because of curry-in-mind API? Or maybe it should depends on order of stream declarations?